intranet/apps/polls/models.py

```diff
@@ -94,6 +94,32 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        groups = DjangoGroup.objects.none()
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return len(users)
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        if able == 0:  # there are no eligible voters
+            return "0.0%"
```

Comment:
I would not return immediately here, and save `0.0` to percentage, so that, if the format of the returned percentage needs to change, it can be done by changing one line.